pom.xml

```diff
@@ -47,6 +47,7 @@
         <module>p2-profile</module>
         <module>distribution</module>
         <module>integration</module>
+        <module>mi_docker_tests</module>
```

Comment:
Are we going to run this in normal profile , what if the docker is not installed.